run.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+from flask import Flask
+
+app = Flask(__name__)
+
+
+@app.route('/')
+def hello_world():
+    return "Hello World!"
```

Comment:
I would prefer that it said "Hello World" using a Jinja template, given that's the tech stack we're going to be using... not writing HTML into the HTTP response directly

Reply:
Is that necessary? The aim of the ticket was just to stand _something_ up as a placeholder for the ops UI.

Reply:
I don't think it adds much effort but happy to recant if @LeoKHoward doesn't want to do it

Reply:
Tried finding the solution but couldn't find a clear cut one so just gonna leave it for now